src/hdmf/backends/hdf5/h5tools.py

```diff
@@ -364,7 +364,9 @@ def copy_file(self, **kwargs):
              'default': True},
             {'name': 'herd', 'type': 'hdmf.common.resources.HERD',
              'doc': 'A HERD object to populate with references.',
-             'default': None})
+             'default': None},
+            {'name': 'expandable', 'type': bool, 'default': True,
+             'doc': 'Bool to set whether datasets are expandable through chunking by default.'})
```

Comment:
Would be nice to make it more explicit in the docstring what is happening under the hood, maybe say     `'doc': 'Bool to set whether datasets are expandable by setting the max shape for all dimensions of a dataset to None and enabling auto-chunking by default.'})`